src/Compilers/CSharp/Test/Semantic/FlowAnalysis/FlowDiagnosticTests.cs

```diff
@@ -1251,11 +1253,17 @@
 }";
             CreateCompilation(program)
                 .VerifyDiagnostics(
-                // (5,12): error CS0843: Backing field for automatically implemented property 'Program.X' must be fully assigned before control is returned to the caller. Consider calling the default constructor from a constructor initializer.
-                Diagnostic(ErrorCode.ERR_UnassignedThisAutoProperty, "Program").WithArguments("Program.X"));
+                 // (7,12): error CS0843: Auto-implemented property 'Program.X2' must be fully assigned before control is returned to the caller.
+                 //     public Program(int x)
+                 Diagnostic(ErrorCode.ERR_UnassignedThisAutoProperty, "Program").WithArguments("Program.X2").WithLocation(7, 12),
+                 // (7,12): error CS0843: Auto-implemented property 'Program.X' must be fully assigned before control is returned to the caller.
+                 //     public Program(int x)
+                 Diagnostic(ErrorCode.ERR_UnassignedThisAutoProperty, "Program").WithArguments("Program.X").WithLocation(7, 12)
+   );
         }
 
         [Fact]
+        [Fact(Skip = "PROTOTYPE(semi-auto-props)")]
```

Comment:
>[Fact(Skip = "PROTOTYPE(semi-auto-props)")] [](http://example.com/codeflow?start=8&length=43)

From the proposed plan for the initial scoped down PR: "Let’s try to avoid disabling any of the legacy tests. Update the baseline, if we don’t like the change in behavior, add a PROTOTYPE comment." We want to be sure the legacy scenarios are not broken. If they are, we want to see how exactly they are broken and whether it is expected given the goals we set for the PR.

Reply:
@AlekseyTs I'm going to restore `FlowDiagnosticTests` as is, with a PROTOTYPE to test `field` keyword again. The behavior of definite assignment is not yet determined actually.

Reverting (IMO) will make the future reviews easier since changes will be clear.